api_calls_string.py

```diff
@@ -0,0 +1,455 @@
+import http.client
+import json
+import datetime
+import os.path
+import csv
+
+
+
+
+api_key = 'ry27AHhE2O4WbIO24c0DE4Zt1KeMvyMx5f31taX1'
+connection = http.client.HTTPSConnection("api.sportradar.us")
+comConnect = http.client.HTTPSConnection("api.sportradar.com")
+
+month = datetime.datetime.now().month
+day = datetime.datetime.now().day
+year = datetime.datetime.now().year
+
+season_year = 2023
+season_type = "REG"
+
+teams_file = "teams.csv"
+
+schedule_file = "game_schedule.csv"
+
+player_list = "All_players_ID.csv"
+
+
+
+# makes a csv file containing the team ids
+def get_teams():
+    #if teams_file exists, exit (and save an api call)
+    if os.path.isfile(teams_file):
+        return None
+    else:
+        try:
+            # Connection and response status 
+            connection.request("GET", f"/nba/trial/v8/en/seasons/{season_year}/{season_type}/rankings.json?api_key={api_key}")
+            response = connection.getresponse()
+    
+            if response.status != 200:
+                print("Error: ", response.status, response.reason)
+                return None
+    
+            # Read & decode data
+            data = response.read()
+            json_data = json.loads(data.decode("utf-8"))
+
+            # creates file to store teams
+            teams = open(teams_file,"w")
+            writer = csv.writer(teams)
+            writer.writerow(["Team ID", "Market", "Team Name"])

```

Comment:
I would combine market and team name in one line as you might have issues in future if mixing up.